src/js/plugins/monitor.js

```diff
@@ -33,14 +33,14 @@ class Monitor extends Plugin {
     }
 
     getMemoryInfo(plugin) {
-       const _rest = {
-            method  : 'GET',
-            path    : 'Monitor'
+        const _rest = {
+            method: 'GET',
+            path: 'Monitor'
```

Comment:
No need to apply code styling changes, please keep the original code that you are not touching as-is and any new code should be written in the code style of the original file. To avoid creating a mixture of code styles and people overwriting their "preferred" style. This creates very convoluted change sets and obstructs the view of the actual code changes applicable to the Pull Request.